qiskit_experiments/framework/restless_mixin.py

```diff
@@ -95,6 +95,7 @@ def enable_restless(
         """
         try:
             if not rep_delay:
+                # BackendV1 only; BackendV2 does not support this
```

Comment:
I'm a little confused. Since `rep_delay_range` is not part of `BackendV2` interface, I don't know where I should look for it. Should I check the existence of a `rep_delay_range` field in the `BackendV2` object? Can I be sure it won't have another name, such as `rep_delay`? Is there a documentation discussing the standard for `BackendV2` fields that might be missing?

Reply:
I guess such value is provided through `backend.options`. One of the important concepts of the backend V2 is the flattened configuration fields (everything is under options), and less backend-specific configuration. So IBM-ish configurations are now removed from the base class. However, you can still assume such configuration exists for a particular experiment relying on it. If the option is not reported, it simply means you cannot run the experiment on that backend. In this sense, it would be better to define dedicated error such as `MissingBackendOptionError` to make this context clearer.

Reply:
@mtreinish please correct me if I'm wrong.

Reply:
Or maybe `rep_delay_range` is directly attached to an IBM backend object as an instance property, because this is not configurable value and `Options` doesn't support static (read-only) field.

Reply:
What confuses me here is that in `BackendV1` the `rep_delay_range` value is in the configuration, not the `Options`, so why would it be there in `BackendV2`? Other fields from the configuration are now present directly as properties of `BackendV2`, sometimes with different name, e.g.
`backend.configuration().max_experiments` ->`backend.max_circuits`.